### PR TITLE
[feat] Add all the attributes of run in the tooltip

### PR DESCRIPTION
**Pull request by VkoHov**
- Add all the attributes of run in the tooltip popover
- Preserve the search input value in the select form autocomplete
- Formatting date values in the tooltip and titles of chart

### CODE REVIEW

**Comment by VkoHov** [comment]
Resolves [#1768](https://github.com/aimhubio/aim/issues/1768)

File: aim/web/ui/src/pages/Metrics/components/SelectForm/SelectForm.tsx [hunk 1]
```diff
@@ -39,6 +39,7 @@ function SelectForm({
   onSearchQueryCopy,
 }: ISelectFormProps): React.FunctionComponentElement<React.ReactNode> {
   const [anchorEl, setAnchorEl] = React.useState<any>(null);
+  const [searchValue, setSearchValue] = React.useState<any>('');
```
Comment (KaroMourad) [inline aim/web/ui/src/pages/Metrics/components/SelectForm/SelectForm.tsx:42]: any to string

File: aim/web/ui/src/pages/Params/components/SelectForm/SelectForm.tsx [hunk 2]
```diff
@@ -29,6 +29,7 @@ function SelectForm({
   selectFormData,
 }: ISelectFormProps): React.FunctionComponentElement<React.ReactNode> {
   const [anchorEl, setAnchorEl] = React.useState<any>(null);
+  const [searchValue, setSearchValue] = React.useState<any>('');
```
Comment (KaroMourad) [inline aim/web/ui/src/pages/Params/components/SelectForm/SelectForm.tsx:32]: the same type-missing here

File: aim/web/ui/src/pages/Params/components/SelectForm/SelectForm.tsx [hunk 3]
```diff
@@ -89,8 +90,21 @@ function SelectForm({
       anchorEl.focus();
     }
     setAnchorEl(null);
+    setSearchValue('');
   }
 
+  function handleSearchInputChange(e: any): void {
```
Comment (KaroMourad) [inline aim/web/ui/src/pages/Params/components/SelectForm/SelectForm.tsx:96]: also here

File: aim/web/ui/src/pages/RunDetail/ImagesVisualizer/ImagesVisualizer.tsx [hunk 4]
```diff
@@ -135,6 +135,7 @@ function ImagesVisualizer(
             additionalProperties={additionalProperties}
             onActivePointChange={onActivePointChange}
             illustrationConfig={{ title: 'No Tracked Images' }}
+            selectOptions={[]}
```
Comment (KaroMourad) [inline aim/web/ui/src/pages/RunDetail/ImagesVisualizer/ImagesVisualizer.tsx:138]: maybe we can set `selectOptions` optional and set an empty array in the component when destructuring the props

**Pull request merged**